lib/smart_answer/date_range.rb

```diff
@@ -72,7 +71,7 @@ def infinite?
     end
 
     def empty?
-      number_of_days == 0
+      number_of_days <= 0
```

Comment:
How about `number_of_days.positive?` ?

Reply:
👍

Reply:
@alecgibson or did you mean `!number_of_days.positive?`?